test/compflags/bradc/printstuff/version.goodstart

```diff
@@ -1 +1 @@
- Version 1.12.0
\ No newline at end of file
+ Version 1.13.0
```

Comment:
I think the newly added linefeed at the end of this line may break this test (or perhaps another that depends on it?)  I.e., I think the lack of a linefeed in the previous version was intentional.